Document/0x05d-Testing-Data-Storage.md

```diff
@@ -879,6 +879,10 @@ While black-box testing the app, navigate to any screen that contains sensitive
 |---|---|
 | <img src="Images/Chapters/0x05d/1.png" width="500px"/> | <img src="Images/Chapters/0x05d/2.png" width="500px"/> |
 
+On device that supports [`file-based encryption(FBE)`](https://source.android.com/security/encryption/file-based "FBE") snapshots are stored in the `/data/system_ce/<USER_ID>/<IMAGE_FOLDER_NAME>` folder. The name of `<IMAGE_FOLDER_NAME>` depends on vendor but most common names are `snapshots` and `recent_images`. If device is not supporting FBE the `/data/system/<IMAGE_FOLDER_NAME>` folder is used.
```

Comment:
```suggestion
On devices supporting [file-based encryption (FBE)](https://source.android.com/security/encryption/file-based "FBE"), snapshots are stored in the `/data/system_ce/<USER_ID>/<IMAGE_FOLDER_NAME>` folder. `<IMAGE_FOLDER_NAME>` depends on the vendor but most common names are `snapshots` and `recent_images`. If the device doesn't support FBE, the `/data/system/<IMAGE_FOLDER_NAME>` folder is used.
```

Reply:
is this folder also called  `snapshots` or `recent_images`in the second case (no FBE)?

Reply:
I guess it is also depends on vendor. 
I saw only variant with ```/data/system/recent_images``` and also this commit say the same [https://github.com/aosp-mirror/platform_frameworks_base/commit/ef73ee1dd98acfc4a19561367cfc3e4d8bbe06ea](https://github.com/aosp-mirror/platform_frameworks_base/commit/ef73ee1dd98acfc4a19561367cfc3e4d8bbe06ea)

Reply:
@cpholguera are you OK if we will use both folder names as an example in case without FBE?